[xslt] Add deprecation console warning upon usage of XSLT APIs

zakariaridouh opened the pull request:
#### ad77bdfcc670bb4350902c86f2419c9f79b9e4ce
<pre>
[xslt] Add deprecation console warning upon usage of XSLT APIs
<a href="https://bugs.webkit.org/show_bug.cgi?id=303881">https://bugs.webkit.org/show_bug.cgi?id=303881</a>
<a href="https://rdar.apple.com/166173246">rdar://166173246</a>

Reviewed by Ryosuke Niwa.

Add a console warning whenever any XSLT code is used by web developers. This shows up in the WebInspector console.

* LayoutTests/fast/events/xsl-onload-expected.txt:
* LayoutTests/fast/parser/external-entities-in-xslt-expected.txt:
* LayoutTests/fast/parser/xslt-with-html-expected.txt:
* LayoutTests/fast/xsl/default-html-expected.txt:
* LayoutTests/fast/xsl/deprecation-warning/resources/document.xml: Added.
* LayoutTests/fast/xsl/deprecation-warning/resources/transform.xsl: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-once-per-page-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-once-per-page.html: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xml-stylesheet-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xml-stylesheet.html: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xsltprocessor-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xsltprocessor.html: Added.
* LayoutTests/fast/xsl/dtd-in-source-document-expected.txt:
* LayoutTests/fast/xsl/exslt-node-set-expected.txt:
* LayoutTests/fast/xsl/extra-lf-at-end-expected.txt:
* LayoutTests/fast/xsl/generate-id-distinct-elements-unique-value-expected.txt:
* LayoutTests/fast/xsl/generate-id-multiple-calls-same-value-expected.txt:
* LayoutTests/fast/xsl/import-after-comment-expected.txt:
* LayoutTests/fast/xsl/import-non-document-node-expected.txt:
* LayoutTests/fast/xsl/mozilla-tests-expected.txt:
* LayoutTests/fast/xsl/nbsp-in-stylesheet-expected.txt:
* LayoutTests/fast/xsl/sort-unicode-expected.txt:
* LayoutTests/fast/xsl/subframe-location-expected.txt:
* LayoutTests/fast/xsl/transform-to-html-expected.txt:
* LayoutTests/fast/xsl/transform-xhr-doc-expected.txt:
* LayoutTests/fast/xsl/transformToFragment-XML-declaration-expected.txt:
* LayoutTests/fast/xsl/utf8-chunks-expected.txt:
* LayoutTests/fast/xsl/xslt-bad-import-uri-expected.txt:
* LayoutTests/fast/xsl/xslt-contentType-expected.txt:
* LayoutTests/fast/xsl/xslt-doc-enc-expected.txt:
* LayoutTests/fast/xsl/xslt-doc-noenc-expected.txt:
* LayoutTests/fast/xsl/xslt-entity-enc-expected.txt:
* LayoutTests/fast/xsl/xslt-fragment-in-empty-doc-expected.txt:
* LayoutTests/fast/xsl/xslt-max-depth-expected.txt:
* LayoutTests/fast/xsl/xslt-mismatched-tags-in-xslt-expected.txt:
* LayoutTests/fast/xsl/xslt-nested-stylesheets-expected.txt:
* LayoutTests/fast/xsl/xslt-node-set-empty-expected.txt:
* LayoutTests/fast/xsl/xslt-processor-expected.txt:
* LayoutTests/fast/xsl/xslt-processor-template-expected.txt:
* LayoutTests/fast/xsl/xslt-recursion-expected.txt:
* LayoutTests/fast/xsl/xslt-second-level-import-expected.txt:
* LayoutTests/fast/xsl/xslt-string-parameters-expected.txt:
* LayoutTests/fast/xsl/xslt-text-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-script-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-to-fragment-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-to-fragment-no-xml-parsing-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-transformToDocument-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-url-expected.txt:
* LayoutTests/fast/xsl/xslt-xhtml-template-expected.txt:
* LayoutTests/http/tests/misc/location-test-xsl-style-sheet-expected.txt:
* LayoutTests/http/tests/misc/xslt-bad-import-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-allowed-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-img-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-allowed-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-allowed2-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-unaffected-by-style-src-1-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-unaffected-by-style-src-2-expected.txt:
* LayoutTests/http/tests/security/cross-origin-xsl-redirect-BLOCKED-expected.txt:
* LayoutTests/http/tests/security/xss-ALLOWED-xsl-external-entity-xslt-docloader-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-document-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-document-redirect-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-document-securityOrigin-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-external-entity-xslt-docloader-expected.txt:
* LayoutTests/http/tests/xmlviewer/dumpAsText/xsl-stylesheet-expected.txt:
* LayoutTests/http/tests/xsl/xslt-import-delayed-subresource-crash-expected.txt:
* LayoutTests/http/tests/xsl/xslt-transform-with-javascript-disabled-expected.txt:
* LayoutTests/platform/glib/fast/xsl/document-function-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-bad-import-uri-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc-cyr-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc16-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc16to16-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-entity-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-extra-content-at-end-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-import-depth-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-mismatched-tags-in-xslt-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-missing-namespace-in-xslt-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-relative-path-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-transform-to-fragment-no-xml-parsing-crash-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt_unicode-expected.txt:
* LayoutTests/platform/glib/http/tests/misc/xslt-bad-import-expected.txt:
* LayoutTests/platform/glib/http/tests/security/contentSecurityPolicy/xsl-redirect-blocked-expected.txt:
* LayoutTests/platform/glib/http/tests/security/cross-origin-xsl-redirect-BLOCKED-expected.txt:
* LayoutTests/platform/glib/http/tests/security/xss-ALLOWED-xsl-external-entity-xslt-docloader-expected.txt: Added.
* LayoutTests/platform/glib/http/tests/xsl/xslt-import-delayed-subresource-crash-expected.txt:
* LayoutTests/platform/gtk/http/tests/security/xss-DENIED-xsl-external-entity-expected.txt:
* LayoutTests/platform/gtk/http/tests/security/xss-DENIED-xsl-external-entity-redirect-expected.txt:
* LayoutTests/platform/ios/fast/xsl/document-function-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc-cyr-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc16-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc16to16-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-entity-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-import-depth-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-relative-path-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt_unicode-expected.txt:
* LayoutTests/platform/mac/fast/xsl/document-function-expected.txt:
* LayoutTests/platform/mac/fast/xsl/sort-locale-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc-cyr-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc16-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc16to16-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-entity-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-extra-content-at-end-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-import-depth-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-missing-namespace-in-xslt-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-relative-path-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt_unicode-expected.txt:
* LayoutTests/platform/mac/http/tests/security/xss-DENIED-xsl-external-entity-expected.txt:
* LayoutTests/platform/mac/http/tests/security/xss-DENIED-xsl-external-entity-redirect-expected.txt:
* LayoutTests/platform/win/http/tests/misc/xslt-bad-import-expected.txt:
* LayoutTests/platform/win/http/tests/security/cross-origin-xsl-redirect-BLOCKED-expected.txt:
* LayoutTests/platform/wpe/fast/xsl/sort-locale-expected.txt:
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::logXSLTDeprecationWarningIfNeeded):
(WebCore::Document::applyPendingXSLTransformsTimerFired):
* Source/WebCore/dom/Document.h:
* Source/WebCore/xml/XSLTProcessor.cpp:
(WebCore::XSLTProcessor::transformToDocument):
(WebCore::XSLTProcessor::transformToFragment):
* Source/WebCore/xml/XSLTProcessor.h:
* Source/WebCore/xml/XSLTProcessor.idl:

Canonical link: <a href="https://commits.webkit.org/317480@main">https://commits.webkit.org/317480@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/44621b509816b9879cc57c9e690c82e85511d955

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/175276 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/49208 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/168/builds/42989 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/184861 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/136417 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/a6b65deb-1b8d-4390-a607-5e6c4e6b92b4) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/177140 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/50500 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/48891 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/136460 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/136417 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/c8290558-72b7-4fa7-89ef-87a36986a967) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/178233 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/39866 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/157211 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/116938 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/6cd633ec-3038-43ac-9a99-9a6b8ac78dfc) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/38507 "Passed tests") | [✅ 🧪 api-mac-debug](https://ews-build.webkit.org/#/builders/165/builds/36883 "Passed tests") | [⏳ 🧪 jsc-wpe](https://ews-build.webkit.org/#/builders/JSC-Tests-WPE-EWS "Waiting to run tests") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/148930 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/169/builds/36704 "Passed tests") | [✅ 🛠 gtk3-libwebrtc](https://ews-build.webkit.org/#/builders/173/builds/33334 "Built successfully") | | 
| | [✅ 🛠 ios-safer-cpp](https://ews-build.webkit.org/#/builders/174/builds/40895 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/41087 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/187282 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/48875 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/156767 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/145403 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/39154 "Built successfully and passed tests") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/49555 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/170/builds/33326 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/145247 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/40079 "Passed tests") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/121748 "Built successfully") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/115434 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/48061 "Built successfully") | [✅ 🧪 mac-site-isolation](https://ews-build.webkit.org/#/builders/185/builds/11677 "Passed tests") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/47464 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/47694 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/47521 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
EWS run on previous version of this PR (hash https://github.com/WebKit/WebKit/commit/c25bbc69a52a5623775182dc8c44b9c5c0bd0420)<details>

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/134849 "Passed style check") | [⏳ 🛠 ios ](https://ews-build.webkit.org/#/builders/iOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 win ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 ios-apple 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/136718 "Passed tests") | [⏳ 🛠 ios-sim ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-AS-debug ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 win-tests ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/137795 "Passed tests") | [⏳ 🧪 ios-wk2 ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | | ⏳ 🛠 vision-apple 
| | [⏳ 🧪 ios-wk2-wpt ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac-debug ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [  ~~🛠 wpe-cairo-libwebrtc~~](https://ews-build.webkit.org/#/builders/166/builds/2955 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| [⏳ 🛠 🧪 jsc ](https://ews-build.webkit.org/#/builders/JSC-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-ios ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk1 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 gtk ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | | 
| [⏳ 🛠 🧪 jsc-arm64 ](https://ews-build.webkit.org/#/builders/JSC-Tests-arm64-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 gtk-wk2 ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 vision-sim ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-AS-debug-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-gtk ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2-stress ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 playstation ](https://ews-build.webkit.org/#/builders/PlayStation-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv ](https://ews-build.webkit.org/#/builders/tvOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-intel-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv-sim ](https://ews-build.webkit.org/#/builders/tvOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-safer-cpp ](https://ews-build.webkit.org/#/builders/macOS-Safer-CPP-Checks-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 watch ](https://ews-build.webkit.org/#/builders/watchOS-26-Build-EWS "Waiting in queue, processing has not started yet") | | | | 
| | [⏳ 🛠 watch-sim ](https://ews-build.webkit.org/#/builders/watchOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | | | | </details>

webkit-early-warning-system commented:
EWS run on current version of this PR (hash https://github.com/WebKit/WebKit/commit/6f4a31639676ee8b7c76cdaf30fe5dca60b4acbe)<details>

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/134850 "Passed style check") | [⏳ 🛠 ios ](https://ews-build.webkit.org/#/builders/iOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 win ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 ios-apple 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/136719 "Passed tests") | [⏳ 🛠 ios-sim ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-AS-debug ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 win-tests ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/137796 "Passed tests") | [⏳ 🧪 ios-wk2 ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | | ⏳ 🛠 vision-apple 
| | [⏳ 🧪 ios-wk2-wpt ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac-debug ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 wpe-cairo-libwebrtc](https://ews-build.webkit.org/#/builders/166/builds/2956 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| [⏳ 🛠 🧪 jsc ](https://ews-build.webkit.org/#/builders/JSC-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-ios ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk1 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/145060 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| [⏳ 🛠 🧪 jsc-arm64 ](https://ews-build.webkit.org/#/builders/JSC-Tests-arm64-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/2/builds/145060 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| | [⏳ 🛠 vision-sim ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-AS-debug-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-gtk](https://ews-build.webkit.org/#/builders/2/builds/145060 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2-stress ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 playstation ](https://ews-build.webkit.org/#/builders/PlayStation-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv ](https://ews-build.webkit.org/#/builders/tvOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-intel-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv-sim ](https://ews-build.webkit.org/#/builders/tvOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-safer-cpp ](https://ews-build.webkit.org/#/builders/macOS-Safer-CPP-Checks-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 watch ](https://ews-build.webkit.org/#/builders/watchOS-26-Build-EWS "Waiting in queue, processing has not started yet") | | | | 
| | [⏳ 🛠 watch-sim ](https://ews-build.webkit.org/#/builders/watchOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | | | | </details>

webkit-early-warning-system commented:
EWS run on current version of this PR (hash https://github.com/WebKit/WebKit/commit/aef3bbff437169982e7fe9303d919d5097af68bc)<details>

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/134855 "Passed style check") | [⏳ 🛠 ios ](https://ews-build.webkit.org/#/builders/iOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 win ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 ios-apple 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/136724 "Passed tests") | [⏳ 🛠 ios-sim ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-AS-debug ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 win-tests ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/137801 "Passed tests") | [⏳ 🧪 ios-wk2 ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | | ⏳ 🛠 vision-apple 
| | [⏳ 🧪 ios-wk2-wpt ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac-debug ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe-cairo-libwebrtc ](https://ews-build.webkit.org/#/builders/WPE-Cairo-LibWebRTC-Build-EWS "Waiting in queue, processing has not started yet") | | 
| [⏳ 🛠 🧪 jsc ](https://ews-build.webkit.org/#/builders/JSC-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-ios ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk1 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/145065 "Build is in progress. Recent messages:") | | 
| [⏳ 🛠 🧪 jsc-arm64 ](https://ews-build.webkit.org/#/builders/JSC-Tests-arm64-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/2/builds/145065 "Build is in progress. Recent messages:") | | 
| | [⏳ 🛠 vision-sim ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-AS-debug-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-gtk](https://ews-build.webkit.org/#/builders/2/builds/145065 "Build is in progress. Recent messages:") | | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2-stress ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 playstation ](https://ews-build.webkit.org/#/builders/PlayStation-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv ](https://ews-build.webkit.org/#/builders/tvOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-intel-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv-sim ](https://ews-build.webkit.org/#/builders/tvOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-safer-cpp ](https://ews-build.webkit.org/#/builders/macOS-Safer-CPP-Checks-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 watch ](https://ews-build.webkit.org/#/builders/watchOS-26-Build-EWS "Waiting in queue, processing has not started yet") | | | | 
| | [⏳ 🛠 watch-sim ](https://ews-build.webkit.org/#/builders/watchOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | | | | </details>

webkit-early-warning-system commented:
EWS run on current version of this PR (hash https://github.com/WebKit/WebKit/commit/32016e913b441a7b89c5c3b39e6edaadcdbcd5f4)<details>

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/134858 "Passed style check") | [⏳ 🛠 ios ](https://ews-build.webkit.org/#/builders/iOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 win ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 ios-apple 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/136727 "Passed tests") | [⏳ 🛠 ios-sim ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-AS-debug ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 win-tests ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/137804 "Passed tests") | [⏳ 🧪 ios-wk2 ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | | ⏳ 🛠 vision-apple 
| | [⏳ 🧪 ios-wk2-wpt ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac-debug ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 wpe-cairo-libwebrtc](https://ews-build.webkit.org/#/builders/166/builds/2964 "Build is in progress. Recent messages:") | | 
| [⏳ 🛠 🧪 jsc ](https://ews-build.webkit.org/#/builders/JSC-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-ios ](https://ews-build.webkit.org/#/builders/iOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk1 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/145068 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| [⏳ 🛠 🧪 jsc-arm64 ](https://ews-build.webkit.org/#/builders/JSC-Tests-arm64-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/2/builds/145068 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| | [⏳ 🛠 vision-sim ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-AS-debug-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-gtk](https://ews-build.webkit.org/#/builders/2/builds/145068 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2-stress ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 playstation ](https://ews-build.webkit.org/#/builders/PlayStation-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv ](https://ews-build.webkit.org/#/builders/tvOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-intel-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv-sim ](https://ews-build.webkit.org/#/builders/tvOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-safer-cpp ](https://ews-build.webkit.org/#/builders/macOS-Safer-CPP-Checks-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 watch ](https://ews-build.webkit.org/#/builders/watchOS-26-Build-EWS "Waiting in queue, processing has not started yet") | | | | 
| | [⏳ 🛠 watch-sim ](https://ews-build.webkit.org/#/builders/watchOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | | | | </details>

zakariaridouh committed:
[xslt] Add deprecation console warning upon usage of XSLT APIs
https://bugs.webkit.org/show_bug.cgi?id=303881
rdar://166173246

Reviewed by NOBODY (OOPS!).

Add a console warning whenever any XSLT code is used by web developers. This shows up in the WebInspector console.

* LayoutTests/fast/events/xsl-onload-expected.txt:
* LayoutTests/fast/parser/external-entities-in-xslt-expected.txt:
* LayoutTests/fast/parser/xslt-with-html-expected.txt:
* LayoutTests/fast/xsl/default-html-expected.txt:
* LayoutTests/fast/xsl/deprecation-warning/resources/document.xml: Added.
* LayoutTests/fast/xsl/deprecation-warning/resources/transform.xsl: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-once-per-page-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-once-per-page.html: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xml-stylesheet-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xml-stylesheet.html: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xsltprocessor-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xsltprocessor.html: Added.
* LayoutTests/fast/xsl/dtd-in-source-document-expected.txt:
* LayoutTests/fast/xsl/exslt-node-set-expected.txt:
* LayoutTests/fast/xsl/generate-id-distinct-elements-unique-value-expected.txt:
* LayoutTests/fast/xsl/generate-id-multiple-calls-same-value-expected.txt:
* LayoutTests/fast/xsl/import-after-comment-expected.txt:
* LayoutTests/fast/xsl/import-non-document-node-expected.txt:
* LayoutTests/fast/xsl/mozilla-tests-expected.txt:
* LayoutTests/fast/xsl/sort-unicode-expected.txt:
* LayoutTests/fast/xsl/subframe-location-expected.txt:
* LayoutTests/fast/xsl/transform-to-html-expected.txt:
* LayoutTests/fast/xsl/utf8-chunks-expected.txt:
* LayoutTests/fast/xsl/xslt-bad-import-uri-expected.txt:
* LayoutTests/fast/xsl/xslt-doc-enc-expected.txt:
* LayoutTests/fast/xsl/xslt-doc-noenc-expected.txt:
* LayoutTests/fast/xsl/xslt-entity-enc-expected.txt:
* LayoutTests/fast/xsl/xslt-max-depth-expected.txt:
* LayoutTests/fast/xsl/xslt-mismatched-tags-in-xslt-expected.txt:
* LayoutTests/fast/xsl/xslt-nested-stylesheets-expected.txt:
* LayoutTests/fast/xsl/xslt-node-set-empty-expected.txt:
* LayoutTests/fast/xsl/xslt-recursion-expected.txt:
* LayoutTests/fast/xsl/xslt-second-level-import-expected.txt:
* LayoutTests/fast/xsl/xslt-string-parameters-expected.txt:
* LayoutTests/fast/xsl/xslt-text-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-to-fragment-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-to-fragment-no-xml-parsing-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-url-expected.txt:
* LayoutTests/fast/xsl/xslt-xhtml-template-expected.txt:
* LayoutTests/http/tests/misc/location-test-xsl-style-sheet-expected.txt:
* LayoutTests/http/tests/misc/xslt-bad-import-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-allowed-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-img-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-allowed-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-allowed2-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-unaffected-by-style-src-1-expected.txt:
* LayoutTests/http/tests/xmlviewer/dumpAsText/xsl-stylesheet-expected.txt:
* LayoutTests/http/tests/xsl/xslt-transform-with-javascript-disabled-expected.txt:
* LayoutTests/platform/mac/fast/xsl/document-function-expected.txt:
* LayoutTests/platform/mac/fast/xsl/sort-locale-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc-cyr-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc16-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc16to16-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-entity-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-extra-content-at-end-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-import-depth-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-missing-namespace-in-xslt-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-relative-path-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt_unicode-expected.txt:
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::logXSLTDeprecationWarningIfNeeded):
(WebCore::Document::applyPendingXSLTransformsTimerFired):
* Source/WebCore/dom/Document.h:
* Source/WebCore/xml/XSLTProcessor.cpp:
(WebCore::XSLTProcessor::transformToDocument):
(WebCore::XSLTProcessor::transformToFragment):

webkit-early-warning-system commented:
EWS run on current version of this PR (hash https://github.com/WebKit/WebKit/commit/fb35a0856ce40bca57bbf82be082172de3eb26fb)<details>

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/135244 "Passed style check") | [⏳ 🛠 ios ](https://ews-build.webkit.org/#/builders/iOS-26-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/142751 "Build is in progress. Recent messages:Printed configuration; Running apply-patch; Checked out pull request; Running compile-webkit") | [⏳ 🛠 win ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 ios-apple 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/137113 "Passed tests") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/8271 "Build is in progress. Recent messages:OS: Tahoe (26.0), Xcode: 26.0.1") | [⏳ 🛠 mac-AS-debug ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/5/builds/142751 "Build is in progress. Recent messages:Printed configuration; Running apply-patch; Checked out pull request; Running compile-webkit") | [⏳ 🧪 win-tests ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/138190 "Passed tests") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/155/builds/8271 "Build is in progress. Recent messages:OS: Tahoe (26.0), Xcode: 26.0.1") | [⏳ 🧪 api-mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-wpe](https://ews-build.webkit.org/#/builders/5/builds/142751 "Build is in progress. Recent messages:Printed configuration; Running apply-patch; Checked out pull request; Running compile-webkit") | | ⏳ 🛠 vision-apple 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/155/builds/8271 "Build is in progress. Recent messages:OS: Tahoe (26.0), Xcode: 26.0.1") | [⏳ 🧪 api-mac-debug ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 wpe-cairo-libwebrtc](https://ews-build.webkit.org/#/builders/166/builds/3340 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-ios](https://ews-build.webkit.org/#/builders/155/builds/8271 "Build is in progress. Recent messages:OS: Tahoe (26.0), Xcode: 26.0.1") | [⏳ 🧪 mac-wk1 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/145447 "Build is in progress. Recent messages:") | | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/7317 "Build is in progress. Recent messages:OS: Tahoe (26.0), Xcode: 26.0.1; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | [⏳ 🧪 mac-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/2/builds/145447 "Build is in progress. Recent messages:") | | 
| | [⏳ 🛠 vision-sim ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-AS-debug-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-gtk](https://ews-build.webkit.org/#/builders/2/builds/145447 "Build is in progress. Recent messages:") | | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-26-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2-stress ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 playstation ](https://ews-build.webkit.org/#/builders/PlayStation-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/7371 "Build is in progress. Recent messages:OS: Tahoe (26.0), Xcode: 26.0.1; Skipping applying patch since patch_id isn't provided; Checked out pull request; Running compile-webkit") | [⏳ 🧪 mac-intel-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 tv-sim ](https://ews-build.webkit.org/#/builders/tvOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-safer-cpp ](https://ews-build.webkit.org/#/builders/macOS-Safer-CPP-Checks-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🛠 watch ](https://ews-build.webkit.org/#/builders/watchOS-26-Build-EWS "Waiting in queue, processing has not started yet") | | | | 
| | [⏳ 🛠 watch-sim ](https://ews-build.webkit.org/#/builders/watchOS-26-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | | | | </details>

zakariaridouh committed:
[xslt] Add deprecation console warning upon usage of XSLT APIs
https://bugs.webkit.org/show_bug.cgi?id=303881
rdar://166173246

Reviewed by Ryosuke Niwa.

Add a console warning whenever any XSLT code is used by web developers. This shows up in the WebInspector console.

* LayoutTests/fast/events/xsl-onload-expected.txt:
* LayoutTests/fast/parser/external-entities-in-xslt-expected.txt:
* LayoutTests/fast/parser/xslt-with-html-expected.txt:
* LayoutTests/fast/xsl/default-html-expected.txt:
* LayoutTests/fast/xsl/deprecation-warning/resources/document.xml: Added.
* LayoutTests/fast/xsl/deprecation-warning/resources/transform.xsl: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-once-per-page-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-once-per-page.html: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xml-stylesheet-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xml-stylesheet.html: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xsltprocessor-expected.txt: Added.
* LayoutTests/fast/xsl/deprecation-warning/xslt-deprecation-warning-xsltprocessor.html: Added.
* LayoutTests/fast/xsl/dtd-in-source-document-expected.txt:
* LayoutTests/fast/xsl/exslt-node-set-expected.txt:
* LayoutTests/fast/xsl/extra-lf-at-end-expected.txt:
* LayoutTests/fast/xsl/generate-id-distinct-elements-unique-value-expected.txt:
* LayoutTests/fast/xsl/generate-id-multiple-calls-same-value-expected.txt:
* LayoutTests/fast/xsl/import-after-comment-expected.txt:
* LayoutTests/fast/xsl/import-non-document-node-expected.txt:
* LayoutTests/fast/xsl/mozilla-tests-expected.txt:
* LayoutTests/fast/xsl/nbsp-in-stylesheet-expected.txt:
* LayoutTests/fast/xsl/sort-unicode-expected.txt:
* LayoutTests/fast/xsl/subframe-location-expected.txt:
* LayoutTests/fast/xsl/transform-to-html-expected.txt:
* LayoutTests/fast/xsl/transform-xhr-doc-expected.txt:
* LayoutTests/fast/xsl/transformToFragment-XML-declaration-expected.txt:
* LayoutTests/fast/xsl/utf8-chunks-expected.txt:
* LayoutTests/fast/xsl/xslt-bad-import-uri-expected.txt:
* LayoutTests/fast/xsl/xslt-contentType-expected.txt:
* LayoutTests/fast/xsl/xslt-doc-enc-expected.txt:
* LayoutTests/fast/xsl/xslt-doc-noenc-expected.txt:
* LayoutTests/fast/xsl/xslt-entity-enc-expected.txt:
* LayoutTests/fast/xsl/xslt-fragment-in-empty-doc-expected.txt:
* LayoutTests/fast/xsl/xslt-max-depth-expected.txt:
* LayoutTests/fast/xsl/xslt-mismatched-tags-in-xslt-expected.txt:
* LayoutTests/fast/xsl/xslt-nested-stylesheets-expected.txt:
* LayoutTests/fast/xsl/xslt-node-set-empty-expected.txt:
* LayoutTests/fast/xsl/xslt-processor-expected.txt:
* LayoutTests/fast/xsl/xslt-processor-template-expected.txt:
* LayoutTests/fast/xsl/xslt-recursion-expected.txt:
* LayoutTests/fast/xsl/xslt-second-level-import-expected.txt:
* LayoutTests/fast/xsl/xslt-string-parameters-expected.txt:
* LayoutTests/fast/xsl/xslt-text-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-script-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-to-fragment-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-transform-to-fragment-no-xml-parsing-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-transformToDocument-crash-expected.txt:
* LayoutTests/fast/xsl/xslt-url-expected.txt:
* LayoutTests/fast/xsl/xslt-xhtml-template-expected.txt:
* LayoutTests/http/tests/misc/location-test-xsl-style-sheet-expected.txt:
* LayoutTests/http/tests/misc/xslt-bad-import-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-allowed-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-img-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-allowed-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-allowed2-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-redirect-blocked-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-unaffected-by-style-src-1-expected.txt:
* LayoutTests/http/tests/security/contentSecurityPolicy/xsl-unaffected-by-style-src-2-expected.txt:
* LayoutTests/http/tests/security/cross-origin-xsl-redirect-BLOCKED-expected.txt:
* LayoutTests/http/tests/security/xss-ALLOWED-xsl-external-entity-xslt-docloader-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-document-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-document-redirect-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-document-securityOrigin-expected.txt:
* LayoutTests/http/tests/security/xss-DENIED-xsl-external-entity-xslt-docloader-expected.txt:
* LayoutTests/http/tests/xmlviewer/dumpAsText/xsl-stylesheet-expected.txt:
* LayoutTests/http/tests/xsl/xslt-import-delayed-subresource-crash-expected.txt:
* LayoutTests/http/tests/xsl/xslt-transform-with-javascript-disabled-expected.txt:
* LayoutTests/platform/glib/fast/xsl/document-function-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-bad-import-uri-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc-cyr-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc16-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-enc16to16-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-entity-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-extra-content-at-end-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-import-depth-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-mismatched-tags-in-xslt-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-missing-namespace-in-xslt-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-relative-path-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt-transform-to-fragment-no-xml-parsing-crash-expected.txt:
* LayoutTests/platform/glib/fast/xsl/xslt_unicode-expected.txt:
* LayoutTests/platform/glib/http/tests/misc/xslt-bad-import-expected.txt:
* LayoutTests/platform/glib/http/tests/security/contentSecurityPolicy/xsl-redirect-blocked-expected.txt:
* LayoutTests/platform/glib/http/tests/security/cross-origin-xsl-redirect-BLOCKED-expected.txt:
* LayoutTests/platform/glib/http/tests/security/xss-ALLOWED-xsl-external-entity-xslt-docloader-expected.txt: Added.
* LayoutTests/platform/glib/http/tests/xsl/xslt-import-delayed-subresource-crash-expected.txt:
* LayoutTests/platform/gtk/http/tests/security/xss-DENIED-xsl-external-entity-expected.txt:
* LayoutTests/platform/gtk/http/tests/security/xss-DENIED-xsl-external-entity-redirect-expected.txt:
* LayoutTests/platform/ios/fast/xsl/document-function-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc-cyr-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc16-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-enc16to16-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-entity-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-import-depth-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt-relative-path-expected.txt:
* LayoutTests/platform/ios/fast/xsl/xslt_unicode-expected.txt:
* LayoutTests/platform/mac/fast/xsl/document-function-expected.txt:
* LayoutTests/platform/mac/fast/xsl/sort-locale-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc-cyr-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc16-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-enc16to16-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-entity-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-extra-content-at-end-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-import-depth-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-missing-namespace-in-xslt-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt-relative-path-expected.txt:
* LayoutTests/platform/mac/fast/xsl/xslt_unicode-expected.txt:
* LayoutTests/platform/mac/http/tests/security/xss-DENIED-xsl-external-entity-expected.txt:
* LayoutTests/platform/mac/http/tests/security/xss-DENIED-xsl-external-entity-redirect-expected.txt:
* LayoutTests/platform/win/http/tests/misc/xslt-bad-import-expected.txt:
* LayoutTests/platform/win/http/tests/security/cross-origin-xsl-redirect-BLOCKED-expected.txt:
* LayoutTests/platform/wpe/fast/xsl/sort-locale-expected.txt:
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::logXSLTDeprecationWarningIfNeeded):
(WebCore::Document::applyPendingXSLTransformsTimerFired):
* Source/WebCore/dom/Document.h:
* Source/WebCore/xml/XSLTProcessor.cpp:
(WebCore::XSLTProcessor::transformToDocument):
(WebCore::XSLTProcessor::transformToFragment):
* Source/WebCore/xml/XSLTProcessor.h:
* Source/WebCore/xml/XSLTProcessor.idl:

Canonical link: https://commits.webkit.org/317480@main
